schema-derivatives-options.graphql

```diff
@@ -851,15 +875,9 @@ type LiquidityPoolDailySnapshot @entity(immutable: true) {
   " Total number of contracts exercised+expired "
   contractsClosedCount: Int!
 
-  " Daily number of open positions (contracts minted-closed)"
-  dailyOpenPositionCount: Int!
-
   " Total number of open positions (contracts minted-closed) "
   openPositionCount: Int!
 
-  " Daily number of closed positions "
-  dailyClosedPositionCount: Int!
-
```

Comment:
why remove these?

Reply:
Since positions are just taken contracts, those fields are already counted under `dailyContractsTakenCount` and `dailyContractsClosedCount`.